Formula/sqlite.rb

```diff
@@ -36,6 +37,12 @@ class Sqlite < Formula
     sha256 "991b40fe8b2799edc215f7260b890f14a833512c9d9896aa080891330ffe4052"
   end
 
+  resource "regexp" do
+    url "http://localhost/regexp.c", :using => :nounzip
```

Comment:
So just use a particular revision. Out of interest, though, why isn't this included with SQLite itself?